src/main/webapp/js/actions/auth-actions.js

```diff
@@ -25,26 +28,26 @@ export const authError = error => ({
   payload: error,
 });
 
-export const signinUser = ({ username, password }, callback) => (dispatch) => {
-  authClient.owner.getToken(username, password)
-    .then((response) => {
-      dispatch({ type: AUTH_USER });
-      const tokenDecoded = jwtDecode(response.accessToken);
-      dispatch({
-        type: SET_PERMISSIONS,
-        payload: tokenDecoded.authorities,
-      });
+export const signinUser = ({ username, password, savedLogin }, callback) => (dispatch) => {
+  getHash(password, (hash) => {
```

Comment:
I think it might be easier to revert those changes and just check the response code in catch instead of using redux-offline here